client/src/main/java/org/eclipse/hono/client/Command.java

```diff
@@ -85,7 +87,8 @@ private Command(
      *
      * @param message The message containing the command.
      * @param tenantId The tenant that the device belongs to.
-     * @param deviceId The identifier of the device.
+     * @param deviceId The identifier of the device that the command will be sent to. This could be a mapped gateway,
+     *                 the original command target device is given in the message address in that case.
```

Comment:
I do not quite get the meaning of the second sentence. Can you try to rephrase it?

Reply:
Done.